internal/declarative/v2/reconciler_test.go

```diff
@@ -0,0 +1,54 @@
+//nolint:testpackage,lll
+package v2
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/cli-runtime/pkg/resource"
+)
+
+func TestPruneKymaSystem(t *testing.T) {
+	t.Parallel()
+	obj1 := &resource.Info{Object: &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "kube-system"}, TypeMeta: metav1.TypeMeta{Kind: "Namespace"}}}
```

Comment:
this obj1, obj2.... random number name we should prevent.

Reply:
Agreeing on this for production code. Will give the test data some names anyways.